docs/features/minimal-api.md

```diff
@@ -64,7 +65,7 @@ builder.Services.AddNCronJob([RetryPolicy(retryCount: 3)] (JobExecutionContext c
 }, "0 * * * *");
 ```
 
-## Concurrency-Support
+## Concurrency-Support
```

Comment:
I swear to god I don't know why `Cmd+Space` is a thing on MacBooks leading to this "non-printable whitespace" which clusterf**ks markdown.